ale/drivers/dawn_drivers.py

```diff
@@ -24,8 +24,8 @@ def instrument_id(self):
         : str
           instrument id
         """
-        instrument_id = self.label["INSTRUMENT_ID"]
-        filter_number = self.label["FILTER_NUMBER"]
+        instrument_id = super().instrument_id
```

Comment:
I just want to make sure that I am fully understanding how super() is being used here... I noticed in the cassini driver (which I was under the impression is "done"?) the line ```self.id_lookup[super().instrument_id]``` is being used to access the inherited instrument_id value. Was there a reason for the more complex access?